addon.gradle

```diff
@@ -1,5 +1,18 @@
+import org.apache.tools.ant.filters.ReplaceTokens
+
 tasks.withType(ScalaCompile) {
     scalaCompileOptions.forkOptions.with {
         memoryMaximumSize = '4g'
     }
 }
+tasks.named("processResources", ProcessResources).configure {
+    it.inputs.file('src/main/resources/application.conf')
+    it.inputs.property('version', project.version)
+    doLast {
+        copy {
```

Comment:
This is already a copy task, you're basically telling it to copy the file twice and filter it the second time. Use filesMatching() to instead modify the settings for that file's copy once